web/src/pages/GroupAPage/GroupAPage.tsx

```diff
@@ -13,8 +13,10 @@ const GroupAPage = () => {
   const handleNextStep = () => {
 
     // Change page
-    if (step === 2){
-      navigate(routes.groupB(), {replace: true})
+    if (step >= 2){
+      return (
```

Comment:
Same suggestion for the return here, you shouldn't return the value returned by the call to `navigate(...)` but simply `return;` to exist the function.